src/Sylius/Bundle/ApiBundle/spec/Mapper/AddressMapperSpec.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace spec\Sylius\Bundle\ApiBundle\Mapper;
+
+use PhpSpec\ObjectBehavior;
+use Sylius\Component\Core\Model\AddressInterface;
+
+final class AddressMapperSpec extends ObjectBehavior
```

Comment:
Spec for this class looks hideous 😆 It can stay, but I believe a PHPUnit test would be more suitable and nicer to read 🖖